dispatcher.sh

```diff
@@ -1438,9 +1438,10 @@ shutdown_daemon() {
     if test -n "${SHUTDOWN}" -a -n "$(eval echo '${'$DAEMON'_OK}')" ; then
         DAEMON_NAME="$(eval echo '${'$DAEMON'_NAME}')"
         DAEMON_SHUT="$(eval echo '${'$DAEMON'_SHUT}')"
+        DAEMON_SHUT_TIMEOUT=" $(eval echo '${TE_'$DAEMON'_SHUT_TIMEOUT}')"
```

Comment:
In general I dislike such ways to pass parameters etc. It is always hard to track it. I think it should be done via dispatcher.sh option instead.